extensions/mssql/package.json

```
@@ -513,6 +513,11 @@
           "group": "0_query@1",
           "isDefault": true
         },
+        {
+          "command": "mssql.objectProperties",
+          "when": "connectionProvider == MSSQL && serverInfo && !isCloud && nodeType && nodeType == Server && mssql:engineedition != 11",
```

This should be dev-only until it's complete enough for customers
```suggestion
          "when": "connectionProvider == MSSQL && serverInfo && !isCloud && nodeType && nodeType == Server && mssql:engineedition != 11 && isDevelopment",
```